monero/wallet_client.go

```diff
@@ -76,6 +76,46 @@ type WalletClientConf struct {
 	LogPath             string               // optional, default is dir(WalletFilePath)/../monero-wallet-rpc.log
 }
 
+// Fill fills in the optional configuration values (Port, MonerodNodes, MoneroWalletRPCPath,
+// and LogPath) if they are not set.
+func (conf *WalletClientConf) Fill() error {
+	if conf.WalletFilePath == "" {
+		panic("WalletFilePath is a required conf field") // should have been caught before we were invoked
+	}
```

Comment:
This check is still in `NewWalletClient` before it calls `Fill()`. In `NewWalletClient`, you probably want the check to see if the wallet file exists to be after the call to `Fill()`.